lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -150,23 +149,9 @@ class RenderForm extends React.Component {
     return (
       <div className="contact-us-wrapper">
 
-        <StatusAlert
-          alertType="info"
-          open
-          dialog={(
-            <div>
-              <span>Due to the recent increase in interest in online education and edX, we are currently experiencing an unusually high volume of support requests. We appreciate your patience as we work to review each request. Please check the </span>
-              <a
-                href="https://support.edx.org/hc/en-us"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Help Center
-              </a>
-              <span> as many questions may have already been answered.</span>
-            </div>
-          )}
-        />
+        <div className="alert alert-warning" role="alert" style={{ marginBottom: '1rem', padding: '1.5rem', left: '0px', fontSize: '16px', backgroundColor: '#fffaed', color: '#171C29', border: '1px solid #FFD875', borderRadius: '0.3rem' }}>
```

Comment:
Can you add your note on why you're not using Paragon to the code as well? I'm guessing someone will come along and wonder why it's different in the future.

Reply:
Done!